charts/patroni-spilo/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```

Comment:
First line comment... maybe should be part of the whole PR thought.

Is /charts/ the best place for all of this? /devops/ and /infrastructure/ exist - so this will be our third top level devopsy folder. I wonder if we should be putting it into one of the other already existing folders.